lms/djangoapps/philu_overrides/helpers.py

```diff
@@ -88,6 +88,10 @@ def get_course_next_classes(request, course):
         # - Course is already full
         # - Student cannot enroll in course
         active_reg_button = not (registered or is_course_full or not can_enroll)
+        target_class_link_for_staff_user = ""
+        if request.user.is_authenticated() and request.user.is_staff:
+            from openedx.core.djangoapps.timed_notification.core import get_course_first_chapter_link
```

Comment:
please add some comments here to describe the reason, for which we are doing this import here.